shared-ui/lib/responsive.ts

```diff
@@ -1,5 +1,5 @@
 export const breakpoints = {
-  mobile: 320,
+  mobile: 820,
```

Comment:
Why was this changed? This most likely will affect many other components in the repo.